test/FunctionalTests/Microsoft.OData.Client.Tests/DataServiceContextHttpClientHandlerProviderTests.cs

```diff
@@ -80,6 +134,62 @@ private HttpResponseMessage HandleRequest(HttpRequestMessage request)
 
             return response;
         }
+
+        /// <summary>
+        /// Creates a new <see cref="Uri"/> by combining <paramref name="uri"/> with each element of <paramref name="uris"/>
+        /// </summary>
+        /// <param name="uri">The base URI that will be added to</param>
+        /// <param name="uris">The URIs to combine with <paramref name="uri"/></param>
+        /// <returns>The combination of <paramref name="uri"/> and <paramref name="uris"/></returns>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="uri"/> or <paramref name="uris"/> is <see langword="null"/></exception>
+        /// <exception cref="ArgumentException">
+        /// Thrown if <paramref name="uri"/> is not an absolute URI or if one of <paramref name="uris"/> is not a relative URI
+        /// </exception>
+        private static Uri Combine(Uri uri, params string[] uris)

```

Comment:
Having a hard time picturing what a combined URI means. Could you share an example?

Reply:
Does this only combine path segments? Does it merge query options?

Reply:
I was hoping that it would be clear that it's calling `new Uri(Uri, string)` for each element in `uris`, but after playing around with it to add some tests as examples, that constructor overload is not very intuitive at all, so I decided to abandon these methods entirely.